packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -121,10 +177,11 @@ withVersions('express', 'express', version => {
         await agent.use((traces) => {
           const span = traces[0][0]
           expect(span.meta).not.to.haveOwnProperty('_dd.appsec.s.req.body')
+          expect(span.meta).not.to.haveOwnProperty('_dd.appsec.s.res.body')
         })
 
         expect(res.status).to.be.equal(200)
-        expect(res.data).to.be.equal('DONE')
+        expect(res.data).to.be.deep.equal('DONE')
```

Comment:
deep equal a string ? what

Reply:
I didn't realize in the review... I made some test changing the `'DONE'` by and object... and forget to restore. Changing it..